File: 1_Year/AED l/www.beecrowd.com.br/Randon/sequencia_soma.c
#include <stdio.h>

int main () {
    int i=0, s=1, f=1, media=0, aux;

    while (s>0 || f>0)
    {
        scanf("%d %d", &s, &f);
        if (s>0 && f>0)
        {
            if (s>f) 
            {
                aux=s;
                s=f;
                f=aux;
            }
            while (s<=f)
            {
                printf("%d ", s);
                media+=s;
                s++;
            }
            printf("Sum=%d\n", media);
            media=0;
        }
        else 
            return 0;

    }
}

File: 1_Year/CursoJava/sequencia_soma.java
import java.util.Scanner;

public class sequencia_soma {
    public static void main(String[] args) {
        int s=1, f=1, media=0, aux;
        while (s>0 || f>0)
        {
            Scanner scanf = new Scanner(System.in);
            s = scanf.nextInt();
            f = scanf.nextInt();
            scanf.close();
            if (s>0 && f>0)
            {
                if (s>f) 
                {
                    aux=s;
                    s=f;
                    f=aux;
                }
                while (s<=f)
                {
                    System.out.printf("%d ", s);
                    media+=s;
                    s++;
                }
                System.out.printf("Sum=%d\n", media);
                media=0;
            }
            else 
                return;
        }
    }
}
